code/__HELPERS/game.dm

```diff
@@ -227,8 +227,8 @@
 		return hear
 
 	var/list/range = get_hear(R, T)
-	for(var/atom/movable/A in range)
-		hear |= recursive_hear_check(A)
+	for(var/turf/A in range)
+		hear += recursive_hear_check(A)
```

Comment:
How so? No more entries are being added to this list than would be added normally anyway